### PR TITLE
disable drop under CSP

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: src/uiSelectSortDirective.js [hunk 1]
```diff
@@ -31,6 +31,10 @@ uis.directive('uiSelectSort', ['$timeout', 'uiSelectConfig', 'uiSelectMinErr', f
         }
       });
 
+      if (element.data('disallowDrop')) {
```
Comment (graingert) [inline src/uiSelectSortDirective.js]: rather than overriding the drop handlers in the html, simply don't set them.

Comment (Jefiozie) [inline src/uiSelectSortDirective.js]: Is this needed? as I don't see anything in the code that is looking for `ondrop`?

Comment (graingert) [inline src/uiSelectSortDirective.js]: That's because the browser does it

Comment (Jefiozie) [inline src/uiSelectSortDirective.js]: you are right. then we are good to go here 👍

**Comment by graingert** [comment]
@Jefiozie any idea when this might get released?

**Pull request merged**